runtime/bcverify/staticverify.c

```diff
@@ -1820,6 +1831,14 @@ j9bcv_verifyClassStructure (J9PortLibrary * portLib, J9CfrClassFile * classfile,
 						goto _formatError;
 					}
 				}
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES) && defined(J9VM_OPT_NEW_FACTORY_METHOD)
```

Comment:
Doesn't the current VT build (that still uses static `<init>`) return from the above error case at line 1831 ?